app/views/payment/v2/student-license/AddonBodyComponent.vue

```diff
@@ -52,7 +52,7 @@
           <span class="license__addon-info-heading-text">{{ $t('payments.ai_custom_tournament') }}</span>
         </div>
         <div class="license__addon-info-subheading">
-          {{ $t('payments.ai_league_info') }} <a href="https://codecombat.com/league" target="_blank">Learn More.</a>
+          {{ $t('payments.ai_league_info') }} <a href="https://codecombat.com/league" target="_blank" @click.stop>Learn More.</a>
```

Comment:
@mrfinch We should probably use a relative link here, since this will be living on codecombat.com already (but maybe staging.codecombat.com or somerandompartner.codecombat.com).